test/e2e/auth/pod_security_policy.go

```diff
@@ -424,8 +423,8 @@ func restrictedPSPInPolicy(name string) *policy.PodSecurityPolicy {
 }
 
 // restrictedPSP creates a PodSecurityPolicy that is most strict.
-func restrictedPSP(name string) *extensionsv1beta1.PodSecurityPolicy {
-	return &extensionsv1beta1.PodSecurityPolicy{
+func restrictedPSP(name string) *policy.PodSecurityPolicy {
```

Comment:
Use `restrictedPSPInPolicy()` instead.

Reply:
@php-coder Thanks for your review! I removed the old func, e2e tests and renamed func `*InPolicy`. PTAL. Thanks!